### ISSUE TITLE
Rank Math + Oxygen

### ISSUE THREAD

**Issue by stevenorechow** [soflyy/oxygen-bugs-and-features#1169]
Rank Math actually works well with Oxygen. There is just one thing that doesn’t work and it is the fact that Rank Math cannot read the alt text of your images added using Oxygen.

Apart from that one teeny-tiny thing, everything else works great. For that issue, they recommend sharing the API access links to the content analysis in the Rank Math plugin to make content from page builder plugins readable by Rank Math.

Here it is:
https://rankmath.com/kb/content-analysis-api/

He also mentioned that known companies have added the compatibility in only a few hours:
https://rankmath.com/compatibility/divi/
https://rankmath.com/compatibility/elementor/
https://rankmath.com/compatibility/themify-builder/
https://rankmath.com/compatibility/page-builder-framework/
https://rankmath.com/compatibility/flothemes/

Not sure there’s a need to reference the 3rd party plugins doing this, but just incase it’ll help you guys:

https://wordpress.org/plugins/wpdevdesign-rankmath-integration-for-oxygen/

https://wpdevdesign.com/oxygen-rank-math-plugin/

https://oxyplugins.com/downloads/oxy-toolbox/

Oxygen is by far the best page builder for building fast, lightweight websites — integrating natively with Rank Math would let your users use the best SEO plugin on the market as well :)

**Comment by louisreingold** [comment]
I think all the SEO plugins are doing it wrong - rather than trying to read the content out of some visual builder's shortcodes or some plugin's Custom Field or some other plugin's custom DB table,  **they should simply analyze the page as it is displayed on the frontend.** 

This would solve every issue with compatibility with various page builders, whether or not they can read image alt text, etc.

No one would have to do any "integration" - they would just work. 

They'd see exactly what Google sees and could simply analyze that.

**Comment by stevenorechow** [comment]
Thanks for the response Louis.

Is that what SEOPress does? Only reason I ask is if I plan on moving more client sites over to Oxygen, it would probably be a decent idea to find whichever one has the closest integration rather than one that relies (somewhat) on a 3rd party plugin in Rank Math.

**Comment by DjReactor** [comment]
Any updates on this one ? Been waiting for it for quite some time now.

**Comment by stevenorechow** [comment]
> Any updates on this one ? Been waiting for it for quite some time now.

I wouldn't expect anything, it would ben
